lib/edge_rider/preload_associations.rb

```diff
@@ -9,10 +14,15 @@ def preload_associations(*args)
     def preload_associations(*args)
       preloader = ActiveRecord::Associations::Preloader
 
-      if preloader.method_defined?(:preload) # Rails 4
-        preloader.new.preload(*args)
-      else
+      if ActiveRecord::VERSION::MAJOR < 4 # Rails ~> 3.1
```

Comment:
Could you check if `preloader` has a(n instance) method `run` here? Also, I would comment `# Rails 3.1 and 3.2`.